docs/installing.md

```diff
@@ -58,196 +58,197 @@ depending on the package manager of your operating system).
    
     Run all the commands in the following sections as root or using the **sudo** command.
 
-### On Debian and Ubuntu using `apt`
+=== "On Debian and Ubuntu using `apt`"
 
-!!! note
+    !!! note
 
-    On Debian and other systems that use the `apt` package manager, such as Ubuntu, components of Percona Distribution for PostgreSQL 11 can only be installed together with the server shipped by Percona (percona-postgresql-11). If you wish to use Percona Distribution for PostgreSQL, uninstall the PostgreSQL package provided by your distribution (postgresql-11) and then install the chosen components from Percona Distribution for PostgreSQL.
+        On Debian and other systems that use the `apt` package manager, such as Ubuntu, components of Percona Distribution for PostgreSQL 11 can only be installed together with the server shipped by Percona (percona-postgresql-11). If you wish to use Percona Distribution for PostgreSQL, uninstall the PostgreSQL package provided by your distribution (postgresql-11) and then install the chosen components from Percona Distribution for PostgreSQL.
 
 
 
-Install the **percona-postgresql-11** package using the following command.
+    Install the **percona-postgresql-11** package using the following command.
 
-```
-$ sudo apt install percona-postgresql-11
-```
+    ```
+    $ sudo apt install percona-postgresql-11
+    ```
 
-Note that this package will not install the components. Use the following commands to install components’ packages:
+=== "On Red Hat Enterprise Linux and derivatives using `yum`" 
 
-Install `pg_repack`:
+    ####Platform Specific Notes
 
-```
-$ sudo apt install percona-postgresql-11-repack
-```
 
-Install `pgAudit`:
+    >If you intend to install Percona Distribution for PostgreSQL on Red Hat Enterprise Linux v8, disable the ``postgresql``  and ``llvm-toolset``modules:
 
-```
-$ sudo apt install percona-postgresql-11-pgaudit
-```
 
-Install `pgBackRest`:
+    >```
+    >$ sudo dnf module disable postgresql llvm-toolset
+    >```
 
-```
-$ sudo apt install percona-pgbackrest
-```
 
-Install `Patroni`:
+    >On CentOS 7, you should install the ``epel-release`` package:
 
-```
-$ sudo apt install percona-patroni
-```
 
-[Install `pg_stat_monitor`](pg-stat-monitor.md)
+    >```
+    >$ sudo yum -y install epel-release
+    >$ sudo yum repolist
+    >```
 
+    Install the **percona-postgresql-11** package using **yum install**.
 
-Install `PgBouncer`:
+    ```
+    $ sudo yum install percona-postgresql11-server
+    ```
 
-```
-$ sudo apt install percona-pgbouncer
-```
+Note that this package will not install the components. Use the following commands to install components’ packages:
 
-Install `pgAudit-set_user`:
+=== "On Debian and Ubuntu"
```

Comment:
Since the first set of instructions is for Yum, could we have the RHEL set of instructions next instead of Debian. It seems odd to jump from the preceding paragraph.